.travis.yml

```diff
@@ -20,6 +20,10 @@ matrix:
     env: OPTIONS="-race"
   - go: 1.8.2
     env: COVERAGE="true"
+  - go: 1.9.4
+    env: OPTIONS="-race"
+  - go: 1.10
```

Comment:
According to https://github.com/travis-ci/travis-ci/issues/9247 , this needs to be quoted as
```
+ - go: "1.10"
```